docker-hub/creating-companies.md

```diff
@@ -0,0 +1,275 @@
+2. Select **Next** to navigate to the section where connected organizations are listed.
+3. In the **Organizations** drop-down, select the organization to add to the connection.
+4. Select **Next** to confirm or change the default organization and team provisioning.
+5. Review the **Connection Summary** and select **Save**.
+
+### Delete a SSO connection
+
+1. In the **Single Sign-On ConnectionsS**, select the **Action** icon.
+2. Select **Delete** and **Delete Connection**.
+3. Continue with the on-screen instructions.
+
+### Delete a domain with a SSO connection
+
+To delete a domain that's part of an SSO Connection, remove the domain from the connection.
+
+1. In the **Single Sign-On Connection**, select the **Action** icon and **Edit**.
+2. Select **Next** to navigate to the section where the connected domains are listed.
+3. Select the **Remove** icon next to the domain that you want to delete.
+4. Select **Next** to review the associated organizations and default organization and team for provisioning.
+5. Review the **Connection Summary** and select **Save**.
+6. In the **Domain** section, select the **Action** icon for the domain.
+7. Select **Delete** and **Delete Domain** to confirm.
+
+## Enforce SSO
+
+1. In the **Single Sign-On Connection**, select the **Action** icon and **Enforce Single Sign-on**.
+
+    > **Note**
+    >
+    > When you enforce SSO,  all members of your organization with a matching domain must authenticate through your IdP.
+2. Continue with the on-screen instructions and verify that you’ve completed the tasks.
+3. Select **Turn on enforcement** to complete.
+
+## Group mapping
+
+Use groups to automate group provisioning from your identity provider, and these updates will sync with your Docker organizations and teams.
+
+   > **Note**
+   >
+   > Use the same names for the Docker teams as your group names in the IdP to prevent further configuration. In order to sync groups, the group must already exist as a team in Docker.
+
+### Okta
+
+1. In **Okta**, navigate to the directory and select **Group**.
+2. Enable **Push Groups**, and select **Save**.
+
+    > **Note**
+    >
+    > For example, **auacatenet:platform** (your organization:your team). This connects all of your teams in Docker to your groups in Okta.
+
+3. In your group, select **Assign people** to add your users to the group.
+(image goes here)
+4. Navigate to **Applications**, configure your application and select **General**.
+5. Select **Next** and update the value for **Group Attribute Statements** (optional) and filter for **Group Attribute Statements** (update this filter as a best practice).
+
+#### Okta mapping with SCIM enabled
+
+If users are signing in via SSO and SCIM, they’re also provisioned via SCIM.
+
+1. In Okta, select the **Provisioning** tab in the Docker application.
+2. In **Settings**, select **Integration**, and **Edit**.
+3. Enable **Push Groups**, and select **Save**.
+
+       > **Note**
```

Comment:
This note and the following note appear as codeblocks in the preview. I'm not sure why though.

Reply:
Thanks Craig! I'm updated it in my local will push the recent changes